src/modules/app-info/app-info.selector.ts

```diff
@@ -0,0 +1,6 @@
+import { AppInfoModel } from './app-info.model';
+
+export const getVersionNumber = (appInfo: AppInfoModel) => {
+  console.log('APP INFO', appInfo);
```

Comment:
Are you intending to keep this log line?

Reply:
Removing it